content/en/database_monitoring/setup_sql_server/selfhosted.md

```diff
@@ -0,0 +1,195 @@
+Database Monitoring runs as an integration on top of the base Agent ([see benchmarks][1]).
+
+Proxies, load balancers, and connection poolers
+: The Agent must connect directly to the host being monitored. For self-hosted databases, `127.0.0.1` or the socket is preferred. The Agent should not connect to the database through a proxy, load balancer, or connection pooler. While this can be an anti-pattern for client applications, each Agent must have knowledge of the underlying host and should stick to a single host for its lifetime, even in cases of failover. If the Datadog Agent connects to different hosts while it is running, the values of metrics will be incorrect.
+
+Data security considerations
+: See [Sensitive information][2] for information about what data the Agent collects from your databases and how to ensure it is secure.
+
+## Configure SQL Server settings
+
+1. Ensure that your SQL Server instance supports SQL Server authentication by enabling `SQL Server and Windows Authentication mode` in the server properties:
+
+_Server Properties_ -> _Security_ -> _SQL Server and Windows Authentication mode_
+
+
+2. Ensure that your SQL Server instance is listening on a specific fixed port. By default, named instances and SQL Server Express are configured for dynamic ports. See [Microsoft's documentation][3] for more details.
+
+### Linux installation
+
+Extra configuration steps are required to monitor SQL Server running on a Linux host:
+
+1. Install an ODBC SQL Server driver, for example the [Microsoft ODBC driver][4].
+2. Copy the `odbc.ini` and `odbcinst.ini` files into the `/opt/datadog-agent/embedded/etc` folder.
+
+## Grant the Agent access
+
+The Datadog Agent requires read-only access to the database server in order to collect statistics and queries.
+
+Create a read-only login to connect to your server:
+
+```SQL
+CREATE LOGIN datadog WITH PASSWORD = '<PASSWORD>';
+CREATE USER datadog FOR LOGIN datadog;
+GRANT SELECT on sys.dm_os_performance_counters to datadog;
+GRANT VIEW SERVER STATE to datadog;
+```
+
+To collect file size metrics per database, ensure the user you created (`datadog`) has [connect permission][5] access to your databases by running:
+
+```SQL
+GRANT CONNECT ANY DATABASE to datadog;
+```
+
+Additionally to collect AlwaysOn and `sys.master_files` metrics, grant the following additional permission:
+
+```SQL
+GRANT VIEW ANY DEFINITION to datadog;
+```
+
+## Install the Agent
+
+If you haven't already installed the Agent for your SQL Server database host, see the [Agent installation instructions][6].
+
+1. Edit the Agent's `conf.d/sqlserver.d/conf.yaml` file to point to your `host` / `port` and set the hosts to monitor. See the [sample sqlserver.d/conf.yaml][7] for all available configuration options.
+
+{{< tabs >}}
+{{% tab "Windows" %}}
```

Comment:
In this instance, the tabs are breaking, and this is a very small blurb from each that I think we can display both at once without the tabs, especially since we haven't split them up earlier on in the page when there were extra steps for linux.

```suggestion
   **Windows:**
```